plugins/woocommerce/src/Blocks/BlockTypes/AddToCartForm.php

```diff
@@ -70,15 +70,18 @@ protected function enqueue_assets( $attributes, $content, $block ) {
 	 * Add increment and decrement buttons to the quantity input field.
 	 *
 	 * @param string $product add-to-cart form HTML.
+	 * @param string $product_name Product name.
 	 * @return stringa add-to-cart form HTML with increment and decrement buttons.
 	 */
-	private function add_steppers( $product ) {
+	private function add_steppers( $product, $product_name ) {
 		// Regex pattern to match the <input> element with id starting with 'quantity_'.
 		$pattern = '/(<input[^>]*id="quantity_[^"]*"[^>]*\/>)/';
 		// Replacement string to add button BEFORE the matched <input> element.
-		$minus_button = '<button type="button" data-wc-on--click="actions.removeQuantity" class="wc-block-components-quantity-selector__button wc-block-components-quantity-selector__button--minus">-</button>$1';
+		/* translators: %s refers to the item name in the cart. */
+		$minus_button = '<button aria-label="' . esc_html( sprintf( __( 'Reduce quantity of %s', 'woocommerce' ), $product_name ) ) . '"type="button" data-wc-on--click="actions.removeQuantity" class="wc-block-components-quantity-selector__button wc-block-components-quantity-selector__button--minus">-</button>$1';
 		// Replacement string to add button AFTER the matched <input> element.
-		$plus_button = '$1<button type="button" data-wc-on--click="actions.addQuantity" class="wc-block-components-quantity-selector__button wc-block-components-quantity-selector__button--plus">+</button>';
+		/* translators: %s refers to the item name in the cart. */
+		$plus_button = '$1<button  aria-label="' . esc_html( sprintf( __( 'Increase quantity of %s', 'woocommerce' ), $product_name ) ) . '" type="button" data-wc-on--click="actions.addQuantity" class="wc-block-components-quantity-selector__button wc-block-components-quantity-selector__button--plus">+</button>';
```

Comment:
There is a double space here:
```suggestion
		$plus_button = '$1<button aria-label="' . esc_html( sprintf( __( 'Increase quantity of %s', 'woocommerce' ), $product_name ) ) . '" type="button" data-wc-on--click="actions.addQuantity" class="wc-block-components-quantity-selector__button wc-block-components-quantity-selector__button--plus">+</button>';
```

Reply:
Fixed with ead6005b8a950db289589d35d3a41934b42d2e3d.